### PR TITLE
Generate token list periodically and post to S3

### PR DESCRIPTION
**Pull request by anxolin**
Schedules a chron job which generates the cow.fi token lists and uploads to S3

This is related to the change on cow.fi where it will load dynamically the list, and it will pre-generate the pages. 
https://github.com/cowprotocol/cow-fi/pull/93

## Test
See execution, takes a long time to complete:
https://github.com/cowprotocol/token-lists/actions/runs/6798553013/job/18482929782

The result is uploaded to:
https://files.cow.fi/tokens/cowFi-tokens.json

### CODE REVIEW

**Comment by anxolin** [comment]
Merging to test the chron job asap. Pls still review it

File: .github/workflows/cowFi-tokens.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v2
+        with:
+          node-version: lts/*
+
+      - name: Set output of cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
```
Comment (alfetopito) [inline .github/workflows/cowFi-tokens.yml:28]: Deprecated way of setting output vars.
Also, you can cache yarn output in the setup node step.
See:

https://github.com/cowprotocol/cowswap/blob/fb24d411a5df88084117f4eb27fd0500db495851/.github/workflows/ci.yml#L29-L50

File: .github/workflows/cowFi-tokens.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: cowFi-tokens.json
+          path: src/cowFi/cowFi-tokens.json
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::693696263829:role/token-list-github-action-role
```
Comment (alfetopito) [inline .github/workflows/cowFi-tokens.yml:88]: Shouldn't this be in a repo variable or secret?

Comment (anxolin) [reply .github/workflows/cowFi-tokens.yml:88]: Maybe? its not pretty, but i don't think has a security implication and is more straight fw to review (secrets are not possible to retrieve in GH, so you can't know for sure the value that someone set)

I leave this one to @ahhda

Comment (alfetopito) [reply .github/workflows/cowFi-tokens.yml:88]: Fair enough

File: .github/workflows/cowFi-tokens.yml (hunk 1 above)
Comment (anxolin) [reply .github/workflows/cowFi-tokens.yml:28]: https://github.com/cowprotocol/token-lists/pull/304

**Pull request merged**